_posts/2022-08-27-ownership-in-an-age-of-microservices.md

```diff
@@ -7,63 +7,63 @@ tags: [Management, Microservices, Tech, Resilience]
 
-While there are some great discussions about when  microservice are or are not a [good fit](https://www.martinfowler.com/bliki/MicroservicePremium.html), with recent examples of [Segment, moving back to a monolith](https://segment.com/blog/goodbye-microservices/). Here, I am going to focus on some of the unique challenges of ownership in microservices and some of the approaches where companies are making larger macro-services (these tend to be aggregation services focused on a specific domain) to help address the issue. For example, [Uber working towards macro-services with a Domain-Oriented Microservice Architecture](https://www.uber.com/blog/microservice-architecture/). One of the common problems called out as a pain point with microservices is ownership, both the Uber example and the [macro-service Domain aggregation evolution at AirBnB](https://medium.com/qe-unit/airbnbs-microservices-architecture-journey-to-quality-engineering-d5a490e6ba4f) are aiming to specifically improve the clarity and ownership of domains of data and the teams and services that support them.
+While there are many great resources out there that discuss microservices, where they may or may not be a [good fit](https://www.martinfowler.com/bliki/MicroservicePremium.html), and how some companies have decided to [move away from them, and back to a monolith](https://segment.com/blog/goodbye-microservices/), in this post I am going to focus on some of the unique challenges that microservices present with regard to **ownership**. Ownership is frequently called out as a common problem with microservices, and both [Uber, working towards macro-services with a Domain-Oriented Microservice Architecture](https://www.uber.com/blog/microservice-architecture/), and [AirBnb's journey towards macro-service Domain aggregation](https://medium.com/qe-unit/airbnbs-microservices-architecture-journey-to-quality-engineering-d5a490e6ba4f) are aimed at improving the clarity of ownership of domains of data and the teams and services that support them.
 
-# microservices and evolving models of ownership
+# Microservices and evolving models of ownership
 
-There are many different models of ownership, and one of the reasons microservices took off was to avoid the tragedy of the commons that often occurs in large shared codebases. We break the systems apart to try to avoid shared codebases that tend to degrade over time. As companies moved to many smaller systems a number of different ownership models emerged:
+There are many different models of ownership, and one of the reasons microservices took off was to avoid the "tragedy of the commons" problem that often occurs in large shared codebases. We break the systems apart in an attempt to avoid shared codebases that tend to degrade over time. As companies moved to many smaller systems, a number of different ownership models emerged:
 
-* Single Owner
-* Orphan codebase
-* Modular Monolith
+* Single Owner (is this a single dev?)
```

Comment:
Not sure if this means a single developer owns a thing?

Reply:
yeah would `single person owner` make that more clear

Reply:
yep, I think so. Will make the change